<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-QLTKSRN" name="testcases.LoginPageTest" tests="5" failures="1" timestamp="08 Nov 2020 06:45:03 GMT" time="50.835" errors="0">
  <testcase name="loginPageTitleTest" time="8.471" classname="testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="loginPageTitleTest" time="9.372" classname="testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected:&lt;CRMPRO - CRM software for customer relationship management, sales, and support.&gt; but was:&lt;CRMPR - CRM software for customer relationship management, sales, and support.&gt;">
      <![CDATA[java.lang.AssertionError: expected:<CRMPRO - CRM software for customer relationship management, sales, and support.> but was:<CRMPR - CRM software for customer relationship management, sales, and support.>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at testcases.LoginPageTest.loginPageTitleTest(LoginPageTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
    </failure>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="loginPageTitleTest" time="8.594" classname="testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="LogoImage" time="8.387" classname="testcases.LoginPageTest"/>
  <testcase name="loginTest" time="16.011" classname="testcases.LoginPageTest"/>
</testsuite> <!-- testcases.LoginPageTest -->
